src/Mod/Sketcher/App/planegcs/GCS.h

```diff
@@ -230,7 +230,7 @@ namespace GCS
         void removeConstraint(Constraint *constr);
 
         // basic constraints
-        int addConstraintEqual(double *param1, double *param2, int tagId=0, bool driving = true);
+        int addConstraintEqual(double *param1, double *param2, int tagId=0, bool driving = true, bool internalalignment = false);
```

Comment:
I've been keeping an eye out for issues of "boolean blindness" recently after @wwmayer introduced me to the idea -- I notice when you actually set this variable to true you are even including a comment to indicate what it is, and I wonder if the `enum` solution might be a good approach here instead.

Reply:
Actually, I just saw the driving boolean and run in autopilot. 

I think you are right. For both of them. I will review and revert.

Reply:
I decided not to change the driving boolean at this time, but only change the internal alignment to an enum, so as not to interfere with the PR of BSpline improved solver support. That can be done separately after merging that other PR in a dedicated PR.